src/main/java/com/mongodb/jdbc/MongoDriver.java

```diff
@@ -126,10 +159,9 @@ public Connection connect(String url, Properties info) throws SQLException {
                     "unexpected driver property info prompt returned: "
                             + String.join(", ", propertyNames));
         }
-        return createConnection(p.left(), info);
     }
 
-    private MongoConnection createConnection(ConnectionString cs, Properties info)
+    private MongoConnection createDialectConnection(ConnectionString cs, Properties info)
```

Comment:
Can we keep the name "createConnection"?
There will be new settings extracted and passed down to create a MongoConnection for the logging.